mapApp/forms/incident.py

```diff
@@ -67,6 +67,7 @@ class IncidentForm(forms.ModelForm):
     def is_valid(self):
 
         # run default, parent validation first
+        # if failed, get results with super(IncidentForm, self).errors
```

Comment:
I'm not quite sure what you mean by this comment. Is it a TODO?

Reply:
looks like I left in a note to myself when I was trying to figure out why the validation in the test wasn't working. I think it could be helpful for someone else though, does this make more sense?

> To debug a failing validation, review the errors stored in `super(IncidentForm, self).errors`